### PR TITLE
basket widget configurable fields

### PR DESCRIPTION
**Pull request by cjohannsen**
### All changes meet the following requirements
- [x] Changelog entry was added
- [x] Changes have been tested
- [x] Plugin can be built

@plentymarkets/ceres-io

### CODE REVIEW

File: resources/js/src/app/components/basket/list/BasketListItem.js [hunk 1]
```diff
@@ -8,13 +8,32 @@ import { mapState } from "vuex";
...
+    props:
+    {
+        template:
+        {
+            type: String,
+            default: "#vue-basket-list-item"
+        },
+        basketItem:
+        {
+            type: Object
```
Comment (lks-mtzn) [inline resources/js/src/app/components/basket/list/BasketListItem.js]: Kurzschreibweise.

File: resources/lang/de/Widget.properties [hunk 2]
```diff
@@ -489,6 +489,11 @@ addressFieldStateId = Bundesland
 addressFieldTitle = Titel
 addressFieldVatNumber = Ust.-Nr.
 ; Basket
+basketBasketDetailsDataAvailability = "Verfügbarkeit"
```
Comment (lks-mtzn) [inline resources/lang/de/Widget.properties]: basketDetails...

File: resources/views/Basket/Components/BasketList.twig [hunk 3]
```diff
@@ -2,17 +2,24 @@
...
                 <div class="h5">{{ trans("Ceres::Template.basketNoItems") }}</div>
             </div>
             <transition-group name="wish-list-item-transition" tag="div">
-                <basket-list-item template="#vue-basket-list-item" v-for="basketItem in basketItems" :key="basketItem.id" :basket-item="basketItem" :size="size" :is-preview="isPreview" language="{{ services.sessionStorage.getLang() }}"></basket-list-item>
+                <basket-list-item
+                    v-for="basketItem in basketItems"
+                    :key="basketItem.id"
+                    :basket-item="basketItem"
+                    :is-preview="isPreview"
+                    language="{{ services.sessionStorage.getLang() }}"
```
Comment (lks-mtzn) [inline resources/views/Basket/Components/BasketList.twig]: Mach das mal über `App.language`. Musst das dann eig. auch garnicht reingeben.

**Review by lks-mtzn — CHANGES_REQUESTED** [review]

File: resources/views/Basket/Components/BasketList.twig (hunk 3 above)
Comment (cjohannsen) [reply resources/views/Basket/Components/BasketList.twig]: https://github.com/plentymarkets/plugin-ceres/pull/1434/commits/2863789a886f16b6d1db8cf610f046675440dce0

File: resources/js/src/app/components/basket/list/BasketListItem.js (hunk 1 above)
Comment (cjohannsen) [reply resources/js/src/app/components/basket/list/BasketListItem.js]: https://github.com/plentymarkets/plugin-ceres/pull/1434/commits/3332e04b641d0301363b1ffd9dd4e8dfdbe2ac07

File: resources/lang/de/Widget.properties (hunk 2 above)
Comment (cjohannsen) [reply resources/lang/de/Widget.properties]: https://github.com/plentymarkets/plugin-ceres/pull/1434/commits/21790204237978412f5c72c4951054c4a99cc2ad

**Review by lks-mtzn — APPROVED** [review]

**Pull request merged**